sql/src/main/java/io/crate/analyze/AlterBlobTableAnalyzer.java

```diff
@@ -38,11 +39,10 @@
         this.schemas = schemas;
     }
 
-    public AlterBlobTableAnalyzedStatement analyze(AlterBlobTable node, Row parameters) {
+    public AlterBlobTableAnalyzedStatement analyze(AlterBlobTable node, Row parameters, SessionContext sessionContext) {
```

Comment:
Please only require the user if nothing else is required from the sessionContext